examples/systemd/sidekiq.service

```diff
@@ -42,13 +42,19 @@ WatchdogSec=10
 
 WorkingDirectory=/opt/myapp/current
 # If you use rbenv:
-# ExecStart=/bin/bash -lc 'exec /home/deploy/.rbenv/shims/bundle exec sidekiq -e production'
+# ExecStart=/bin/bash -lc 'exec /home/deploy/.rbenv/shims/bundle exec sidekiq -e production' -C config/sidekiq.yml
 # If you use the system's ruby:
-# ExecStart=/usr/local/bin/bundle exec sidekiq -e production
+# ExecStart=/usr/local/bin/bundle exec sidekiq -e production -C config/sidekiq.yml
 # If you use rvm in production without gemset and your ruby version is 2.6.5
-# ExecStart=/home/deploy/.rvm/gems/ruby-2.6.5/wrappers/bundle exec sidekiq -e production
+# ExecStart=/home/deploy/.rvm/gems/ruby-2.6.5/wrappers/bundle exec sidekiq -e production -C config/sidekiq.yml
 # If you use rvm in production with gemset and your ruby version is 2.6.5
-ExecStart=/home/deploy/.rvm/gems/ruby-2.6.5@gemset-name/wrappers/bundle exec sidekiq -e production
+ExecStart=/home/deploy/.rvm/gems/ruby-2.6.5@gemset-name/wrappers/bundle exec sidekiq -e production -C config/sidekiq.yml
+
+# To work with the gem 'capistrano/sidekiq'
+# If you are not using this gem, just remove the two lines just below
+# although it will not have any effect on the service if you keep these two lines
+ExecReload=/bin/kill -TSTP $MAINPID
```

Comment:
I think it’s a bad idea to change the semantics of reload to map it to quiet.

Reply:
I got this from the systemd service template in the "sidekiq/capistrano" repository. The gem uses the reload command to reload the sidekiq service and by default the sidekiq service does not have the "reload" command, it only has the start and stop commands. Para fazer funcionar, peguei "ExecReload" e o adicionei ao serviço sidekiq.

In the link below you can see how the gem uses ExecReload: https://github.com/seuros/capistrano-sidekiq/blob/master/lib/generators/capistrano/sidekiq/systemd/templates/sidekiq.service.capistrano.erb

Reply:
I don't like the idea of mapping `reload` to `quiet`. These two words mean very different things. I would suggest that capistrano-sidekiq move away from reload and use `systemctl kill -TSTP` instead.

Reply:
Ok